### PR TITLE
Author claim for v0.4

### PR DESCRIPTION
**Pull request by louislai**
@Rinder5 @Melvin-Tan I added @@author tags for you guys as well. So check again if anything is missing and modify as necessary

### CODE REVIEW

File: src/main/java/seedu/toluist/commons/util/FileUtil.java [hunk 1]
```diff
@@ -51,6 +51,7 @@ public static boolean removeFile(File file) {
         return file.delete();
     }
 
+    //@@author
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/commons/util/FileUtil.java:54]: No matric number

File: src/main/java/seedu/toluist/controller/AddTaskController.java [hunk 2]
```diff
@@ -1,3 +1,4 @@
+//@@author Melvin
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/controller/AddTaskController.java]: My matric number is A0127545A if u need it :D

File: src/test/java/seedu/toluist/commons/util/CollectionUtilTest.java [hunk 3]
```diff
@@ -160,6 +163,7 @@ public void toSring() {
         assertEquals(CollectionUtil.toString("-", Arrays.asList("a", "b")), "a-b");
     }
 
+    //@@author
```
Comment (Melvin-Tan) [inline src/test/java/seedu/toluist/commons/util/CollectionUtilTest.java:166]: who?

File: src/test/java/seedu/toluist/testutil/TestUtil.java [hunk 4]
```diff
@@ -86,6 +87,22 @@ public static boolean compareTasksOfTodoLists(TodoList todoList1, TodoList todoL
...
+     * @throws IllegalAccessException
+     */
+    public static void resetSingleton(Class klass) throws SecurityException, NoSuchFieldException,
+            IllegalArgumentException, IllegalAccessException {
+        Field instance = klass.getDeclaredField("instance");
+        instance.setAccessible(true);
+        instance.set(null, null);
+    }
+
+    //@@author
```
Comment (Melvin-Tan) [inline src/test/java/seedu/toluist/testutil/TestUtil.java:103]: no author

File: src/main/java/seedu/toluist/controller/AddTaskController.java (hunk 2 above)
Comment (louislai) [reply src/main/java/seedu/toluist/controller/AddTaskController.java]: can u find and replace urself :o

File: src/test/java/seedu/toluist/commons/util/CollectionUtilTest.java (hunk 3 above)
Comment (louislai) [reply src/test/java/seedu/toluist/commons/util/CollectionUtilTest.java:166]: check handbook

**Pull request merged**